<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
    <property name="geometry">
     <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>400</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
        <widget class="QGroupBox" name="groupBox_connection">
         <property name="title">
            <string>Device Connection</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_connection">
            <item row="0" column="0">
             <widget class="QLabel" name="label_device">
                <property name="text">
                 <string>Device Name:</string>
                </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_device"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_cert">
                <property name="text">
                 <string>Certificate File:</string>
                </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_cert"/>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="button_browse_cert">
                <property name="text">
                 <string>Browse</string>
                </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_key">
                <property name="text">
                 <string>Key File:</string>
                </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_key"/>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="button_browse_key">
                <property name="text">
                 <string>Browse</string>
                </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="button_connect">
                <property name="text">
                 <string>Connect</string>
                </property>
             </widget>
            </item>
         </layout>
        </widget>
     </item>
     <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_edit">
            <property name="title">
             <string>Editable Shadow Document</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_edit">
             <item>
              <widget class="QTextEdit" name="textEdit_editable"/>
             </item>
             <item>
              <widget class="QPushButton" name="button_send">
                <property name="text">
                 <string>Send</string>
                </property>
              </widget>
             </item>
            </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_readonly">
            <property name="title">
             <string>Reported Shadow Document</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_readonly">
             <item>
              <widget class="QTextEdit" name="textEdit_readonly">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_refresh">
                <property name="text">
                 <string>Refresh</string>
                </property>
              </widget>
             </item>
            </layout>
          </widget>
         </item>
        </layout>
     </item>
    </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
